app/models/space_host_policy.rb

```diff
@@ -0,0 +1,60 @@
+require 'conjur_client'
+
+class SpaceHostPolicy
+  include ConjurApiModel
+
+  class << self
+    def create(org_id, space_id)
+      SpaceHostPolicy.new(org_id, space_id).create
+    end
+  end
+
+  def initialize(org_id, space_id)
+    @org_id = org_id
+    @space_id = space_id
+  end
+
+  def create
+    result = conjur_api.load_policy(space_policy, template_create, method: Conjur::API::POLICY_METHOD_POST)
+    
+    created_role = result.created_roles.values.first
+
+    if created_role != nil
+      variable_id = "#{space_policy}/#{space_host_api_key_variable_id}"
+      set_variable(variable_id, created_role['api_key'])
+    end
+  end
+
+  private
+
+  def template_create
+    <<~YAML.strip + "\n"
+    - !host
+      id: #{space_host_id}
+
+    - !grant
+      role: !layer
+      member: !host #{space_host_id}
+
+    - !variable
+      id: #{space_host_api_key_variable_id}
+
+    - !permit
+      role: !host /#{ConjurClient.login_host_id}
+      privileges: [read]
+      resource: !variable #{space_host_api_key_variable_id}
+    YAML
+  end
+  
+  def space_host_id
+    "space_host"
+  end
+
+  def space_host_api_key_variable_id
+    "space_host_api_key"
```

Comment:
If we did remove the "space_host" portion, this could become just "api-key".

Reply:
Also, elsewhere we seem to use "dashes" rather than underscores as id word separators.

Reply:
Will switch to dashes but "api-key" is pretty non-descript. I would definitely prefer something shorter but I not to the point of potential confusion.